static/files/download-data-for-wlan-firmwares.bat

```diff
@@ -20,21 +20,28 @@ if /i {%a4%}=={y} goto :yes
 if /i {%a4%}=={yes} goto :yes
 set HAIKU_TEMP=system\data\firmware
 :yes
-set baseURL=http://cgit.haiku-os.org/haiku/plain/src/system/libroot/posix
+set baseURL=https://raw.githubusercontent.com/haiku/firmware/master/wifi
+set haikuPortsURL=https://eu.hpkg.haiku-os.org/haikuports/master/x86_64/current/packages

```

Comment:
This makes the script usable for only one architecture (here x86_64 was chosen). I think there should be a command line switch to allow getting the x86_gcc2 version instead?

This script is more likely to be useful on older hardware, not sure if many 64bit systems need it.

Reply:
Yes, that works too and it's simpler :)